packages/contracts-bedrock/interfaces/L1/IBatchInbox.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import {CertManager} from "@nitro-validator/CertManager.sol";
+
+type CborElement is uint256;
+
+library NitroValidator {
```

Comment:
Maybe add some comments about what Nitro is, specially so that we don't confuse it with the chain.